### PR TITLE
AND-235 Moved strings to RC

### PR DESCRIPTION
**Pull request by vandus**
# Description

Moved a string to RC so it can be changed from remote.

## Screenshots 📸

# How Has This Been Tested? 👨‍🔬

Ran app on dev.

## What Has Not Been Tested? 🙅🏻‍♂️

# Checklist ✅

- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have checked all visual changes in both light and dark mode.

### CODE REVIEW

File: app/src/main/kotlin/cz/covid19cz/erouska/AppConfig.kt [hunk 1]
```diff
@@ -122,6 +122,8 @@ object AppConfig {
         get() = firebaseRemoteConfig.getBoolean("v2_efgsTravellerDefault")
     val howItWorksUITitle
         get() = firebaseRemoteConfig.getString("v2_howItWorksUITitle")
+    val howItWorksEvalContent
```
Comment (davidvavra) [inline app/src/main/kotlin/cz/covid19cz/erouska/AppConfig.kt:125]: When adding new localizable RC key, please follow this guide: https://github.com/covid19cz/erouska-remote-config#how-to-add-new-localizable-rc-key

**Review by davidvavra — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/cz/covid19cz/erouska/AppConfig.kt (hunk 1 above)
Comment (vandus) [reply app/src/main/kotlin/cz/covid19cz/erouska/AppConfig.kt:125]: I am following that guide, I am at no.4

Comment (davidvavra) [reply app/src/main/kotlin/cz/covid19cz/erouska/AppConfig.kt:125]: So go ahead with no. 5 and 6 :)

**Review by davidvavra — APPROVED** [review]
Merge if you want to deal with RC defaults later

**Comment by vandus** [comment]
Agreed offline to add RC defaults before the release

**Comment by Gh0stahCZ** [comment]
@vandac when you are removing string, remove it also from OneSkype (or upload updated strings.xml) otherwise it will get back in next export

**Comment by vandus** [comment]
Sure, you are absolutely right.

**Comment by davidvavra** [comment]
@Gh0stahCZ We agreed that we will not upload strings in pull requests (because it can lead to conflicts) but only on develop before release.

**Comment by Gh0stahCZ** [comment]
@davidvavra we should write these agreements down

**Comment by davidvavra** [comment]
OK, I'm going to write to to README.md

**Pull request merged**